### PR TITLE
Added @Nullable to fluid stuff

### PR DESCRIPTION
**Pull request by laci200270**

### CODE REVIEW

File: src/main/java/net/minecraftforge/fluids/IFluidHandler.java [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package net.minecraftforge.fluids;
 
 import net.minecraft.util.EnumFacing;
+import edu.umd.cs.findbugs.annotations.Nullable;
```
Comment (matthewprenger) [inline src/main/java/net/minecraftforge/fluids/IFluidHandler.java]: Wrong import. Should be: `javax.annotation.Nullable`

File: src/main/java/net/minecraftforge/fluids/TileFluidHandler.java [hunk 2]
```diff
@@ -1,9 +1,11 @@
 
 package net.minecraftforge.fluids;
 
+
 import net.minecraft.nbt.NBTTagCompound;
 import net.minecraft.tileentity.TileEntity;
 import net.minecraft.util.EnumFacing;
+import edu.umd.cs.findbugs.annotations.Nullable;
```
Comment (matthewprenger) [inline src/main/java/net/minecraftforge/fluids/TileFluidHandler.java]: Wrong import

File: src/main/java/net/minecraftforge/fluids/TileFluidHandler.java [hunk 3]
```diff
@@ -31,15 +33,19 @@ public void writeToNBT(NBTTagCompound tag)
 
     /* IFluidHandler */
     @Override
-    public int fill(EnumFacing from, FluidStack resource, boolean doFill)
+    public int fill(@Nullable EnumFacing from, FluidStack resource, boolean doFill)
     {
-        return tank.fill(resource, doFill);
+        if(from==null)
+        	from=EnumFacing.NORTH;
```
Comment (matthewprenger) [inline src/main/java/net/minecraftforge/fluids/TileFluidHandler.java]: Why are you nullchecking these if they're never used in the method?

File: src/main/java/net/minecraftforge/fluids/TileFluidHandler.java [hunk 4]
```diff
@@ -31,15 +33,19 @@ public void writeToNBT(NBTTagCompound tag)
...
+    	return tank.fill(resource, doFill);
```
Comment (Lunatrius) [inline src/main/java/net/minecraftforge/fluids/TileFluidHandler.java]: Mixed spaces/tabs, you have them all over the files. With proper whitespace lines like these wouldn't even appear in the diff.

**Pull request closed without merging**